### ISSUE TITLE
[BUG] SfImage - CLS - explicit width and height can't have the same values as the ones passed to styles

### ISSUE THREAD

**Issue by justyna-13** [vuestorefront/storefront-ui#2117]
**Describe the bug**

Now it's possible to pass props for images like image-height="auto"  image-width="100%". These props are meant to style component image but also are used for explicit width and height 

**How to reproduce**
1. Go to SfProductCard
2. Check img tag

**Expected behavior**
Explicit width and height should have proper values
